library/maven

```diff
@@ -46,27 +46,27 @@ Architectures: amd64, arm32v5, arm32v7, i386
 GitCommit: 05f4802aa5c253dcf75fe967c6f45b3fb1e2f26e
 Directory: jdk-7
 
-Tags: 3.6.0-jdk-8-alpine, 3.6.0-alpine, 3.6-jdk-8-alpine, 3.6-alpine, 3-jdk-8-alpine, alpine
+Tags: 3.6.0-jdk-8-alpine, 3.6-jdk-8-alpine, 3-jdk-8-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
 GitCommit: 05f4802aa5c253dcf75fe967c6f45b3fb1e2f26e
 Directory: jdk-8-alpine
 
-Tags: 3.6.0-jdk-8-slim, 3.6.0-slim, 3.6-jdk-8-slim, 3.6-slim, 3-jdk-8-slim, slim
+Tags: 3.6.0-jdk-8-slim, 3.6-jdk-8-slim, 3-jdk-8-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
 GitCommit: 05f4802aa5c253dcf75fe967c6f45b3fb1e2f26e
 Directory: jdk-8-slim
 
-Tags: 3.6.0-jdk-8, 3.6.0, 3.6-jdk-8, 3.6, 3-jdk-8, 3, latest
+Tags: 3.6.0-jdk-8, 3.6-jdk-8, 3-jdk-8
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
 GitCommit: 05f4802aa5c253dcf75fe967c6f45b3fb1e2f26e
 Directory: jdk-8
 
-Tags: 3.6.0-ibmjava-8-alpine, 3.6.0-ibmjava-alpine, 3.6-ibmjava-8-alpine, 3.6-ibmjava-alpine, 3-ibmjava-8-alpine, ibmjava-alpine
+Tags: 3.6.0-ibmjava-8-alpine, 3.6-ibmjava-8-alpine, 3-ibmjava-8-alpine
 Architectures: amd64
 GitCommit: 05f4802aa5c253dcf75fe967c6f45b3fb1e2f26e
 Directory: ibmjava-8-alpine
 
-Tags: 3.6.0-ibmjava-8, 3.6.0-ibmjava, 3.6-ibmjava-8, 3.6-ibmjava, 3-ibmjava-8, 3-ibmjava, ibmjava
+Tags: 3.6.0-ibmjava-8, 3.6-ibmjava-8, 3-ibmjava-8
```

Comment:
Did you want to lose all these `ibmjava` and `alpine` tags?

```diff
-maven:3-ibmjava
-maven:3.6-alpine
-maven:3.6-ibmjava
-maven:3.6-ibmjava-alpine
-maven:3.6.0-alpine
-maven:3.6.0-ibmjava
-maven:3.6.0-ibmjava-alpine
-maven:alpine
-maven:ibmjava
-maven:ibmjava-alpine
```

Reply:
Friendly ping @carlossg -- was the removal of these tags intentional?  Should we still consider this PR for merging?